src/gui/issueswidget.cpp

```diff
@@ -400,14 +401,9 @@ std::function<void(void)> IssuesWidget::addStatusFilter(QMenu *menu)
     menu->addSeparator();
 
     // Add the other non-error items:
-    const std::vector<SyncFileItem::Status> OtherStatusItems = {
-        SyncFileItem::Status::Conflict,
-        SyncFileItem::Status::FileIgnored,
-        SyncFileItem::Status::Restoration,
-        SyncFileItem::Status::BlacklistedError,
-        SyncFileItem::Status::Excluded,
-        SyncFileItem::Status::Message
-    };
+    const std::vector<SyncFileItem::Status> OtherStatusItems = {SyncFileItem::Status::Conflict, SyncFileItem::Status::FileIgnored,
```

Comment:
somtimes I hate clang format, can you end each line with
`,//`
To force clang format to not do what ever this is... :D

Reply:
done